src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,22 @@
 import React from 'react';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
-);
+import { Post } from '../../types/Post';
+
+import { PostInfo } from '../PostInfo';
+
+type Props = {
+  posts: Post[];
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => {
```

Comment:
You can import `FC` from react as well and avoid reading from object (not critical)